src/markdown-pages/install-with-kurl/setup-tls-certs.md

```diff
@@ -11,14 +11,20 @@ After kURL install has completed, you'll be prompted to set up the KOTS Admin Co
 ![tls-certs-insecure](/tls-certs-insecure.png)
 
 
-The next page allows you to configure your TLS certificates:
+The next page lets you configure your TLS certificates:
 <br><br><br>
 ![tls-certs-setup](/tls-certs-setup.png)
 
-To continue with the preinstalled self-signed TLS certificates, click "skip & continue".  Otherwise upload your signed TLS certificates as describe on this page.  The hostname is an optional field, and when its specified, its used to redirect your browser to the specified host. 
+**Hostname** is a required field that is used when generating self-signed certificates and custom certificates, and redirects your browser to the specified host.
```

Comment:
hostname is an optional field, but it is required when configuring the identity service